flexget/plugins/filter/exists_movie.py

```diff
@@ -74,54 +94,76 @@ def on_task_filter(self, task, config):
             #logging.getLogger('imdb_lookup').setLevel(logging.WARNING)
 
             # scan through
-
-            # TODO: add also video files?
-            for item in folder.walkdirs(errors='ignore'):
-                if item.name.lower() in self.skip:
-                    continue
-                count_dirs += 1
-
-                movie = get_plugin_by_name('parsing').instance.parse_movie(item)
-
-                try:
-                    imdb_id = imdb_lookup.imdb_id_lookup(movie_title=movie.name,
-                                                         raw_title=item.name,
-                                                         session=task.session)
-                    if imdb_id in path_ids:
-                        log.trace('duplicate %s' % item)
+            for root, dirs, files in os.walk(folder):
+                for item in eval(config['type']):
```

Comment:
You can use http://pythonhosted.org/path.py/api.html#path.Path.walk to walk files and dirs at the same time as needed, but since it's one or the other in the config, I'm not sure why we'd need to do that.